components/calendar/src/chinese.rs

```diff
@@ -659,6 +681,56 @@ impl Chinese {
             related_iso,
         }
     }
+
+    /// Get the ordinal lunar month from a code
+    ///
+    /// TODO: The behavior of this fn is calendar specific, but this needs to be implemented in every lunar
+    /// calendar; consider abstracting this function to a Lunar trait
+    fn ordinal_lunar_month_from_code(year: i32, code: MonthCode) -> Option<u8> {
+        if code.0.len() < 3 {
+            return None;
+        }
+        let mid_year = Self::fixed_mid_year_from_year(year);
+        let leap_month = if Self::is_leap_year(year) {
+            Self::get_leap_month_in_year(mid_year)
+        } else {
+            14
```

Comment:
nit: document that this 14 value is there as a sentinel value, greater than all other months

Reply:
Done